### ISSUE TITLE
regression训练速度很慢

### ISSUE THREAD

**Issue by liuhongye216** [modelscope/ms-swift#4294]
{'loss': 0.57576481, 'grad_norm': 111.29665375, 'learning_rate': 9.955e-05, 'memory(GiB)': 15.13, 'train_speed(iter/s)': 0.684019, 'epoch': 0.09, 'global_step/max_steps': '167000/1847700', 'percentage': '9.04%', 'elapsed_time': '2d 19h 49m 4s', 'remaining_time': '28d 10h 31m 31s'}，同样的数据量和算力，训练chat模型大概一周就能练完。

CUDA_VISIBLE_DEVICES=3 \
swift sft \
    --model /home/liuhongye/Model/3.1 \
    --model_type llama3_1 \
    --train_type lora \
    --dataset /home/liuhongye/material_LLM/material_LLM_experiment/LLM_input/mof_regression_train2.jsonl \
    --torch_dtype bfloat16 \
    --num_train_epochs 1 \
    --per_device_train_batch_size 1 \
    --per_device_eval_batch_size 1 \
    --learning_rate 1e-4 \
    --lora_rank 8 \
    --lora_alpha 32 \
    --target_modules all-linear \
    --gradient_accumulation_steps 1 \
    --eval_steps 1000 \
    --save_steps 1000 \
    --save_total_limit 2 \
    --logging_steps 50 \
    --max_length 2048 \
    --output_dir /home/liuhongye/Model/MOF_llama3_1 \
    --warmup_ratio 0.05 \
    --dataloader_num_workers 4 \
    --dataset_num_proc 4 \
    --num_labels 1 \
    --task_type seq_cls \
    --use_chat_template false \
    --problem_type regression

![Image](https://github.com/user-attachments/assets/7199c4cd-81e2-4662-8cc5-5c102919b6e4)

**Comment by liuhongye216** [comment]
在batchsize调大之后会报错ValueError: Cannot handle batch sizes > 1 if no padding token is defined.我的数据集是按照官方文档组织的，格式如下：
{"messages": [{"role": "user", "content": "Please predict the adsorption capacity......"}], "label": 0.014}
{"messages": [{"role": "user", "content": "What is the total uptake of CarbonDioxide at a pressure of 0.1 bar......"}], "label": 0.161}

**Comment by Jintao-Huang** [comment]
> > batch_size调大吧
> > 在batchsize调大之后会报错ValueError: Cannot handle batch sizes > 1 if no padding token is defined.
> > 我的数据集是按照官方文档组织的，格式如下：
> > {"messages": [{"role": "user", "content": "Please predict the adsorption capacity......"}], "label": 0.014}
> > {"messages": [{"role": "user", "content": "What is the total uptake of CarbonDioxide at a pressure of 0.1 bar......"}], "label": 0.161}

fixed
